CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),  
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+
+## 0.0.37 - 2025-05-10
+
+### Added
+
+- Vite plugin integration for Fable projects.
+- Fable.Compiler updated to 5.0.0-alpha.13
+- Added caret range to fable-library-js
```

Comment:
`fable-library-js` really should be tied to our Fable.Compiler version.
Would think `~` makes more sense here.

Reply:
not sure if I understood, pls review updated file, i added also older history if it makes sense